src/states/main_game.rs

```diff
@@ -240,7 +241,26 @@ impl<'a> State<GameData<'a, 'a>, CustomStateEvent> for MainGameState {
                 });
             }
         }
-
+        //insert single nushi
+        {
+            let mut rng = thread_rng();
+            let x = rng.gen_range(left, right);
+            let y = rng.gen_range(bottom, top);
+            let scale = rng.gen_range(0.8f32, 1.2f32);
+            //let rotation = rng.gen_range(0.0f32, PI);
```

Comment:
Remember to remove these comments before merging :). White lines here and there would be nice for separating implicit the blocks of logic (such as the rng related logic, or transform related logic).

Reply:
unnecessary commented lines removed